Java/Trees/Node.java

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Why have you included this file

Reply:
@Kumar-laxmi sir, Done with all the changes,